src/main/java/edu/hawaii/its/api/service/MembershipService.java

```diff
@@ -83,10 +104,17 @@ public MembershipResults membershipResults(String currentUser, String uid) {
     private List<MembershipResult> createMemberships(List<Group> membershipGroupings, List<String> optOutList) {
         List<MembershipResult> memberships = new ArrayList<>();
         for (Group grouping : membershipGroupings) {
+            GroupAttributeResults groupAttributeResults =
+                    grouperService.groupAttributeResult(grouping.getGroupPath());
             MembershipResult membershipResult = new MembershipResult();
             membershipResult.setDescription(grouping.getDescription());
             membershipResult.setPath(grouping.getGroupPath());
             membershipResult.setName(nameGroupingPath(grouping.getGroupPath()));
+            for (int i = 0; i < groupAttributeResults.getGroupAttributes().size(); i++) {
+                if (groupAttributeResults.getGroupAttributes().get(i).getAttributeName().endsWith("curated")) {
+                    membershipResult.setName(grouping.getGroupPath());
+                }
+            }
```

Comment:
There might be a faster way to go about this. Having to loop through all the groupAttributes to check if they are curated adds some time while you already know which are the curated groupings inside the membershipResults function.